### ISSUE TITLE
Is saving too quick?

### ISSUE THREAD

**Issue by karmatosed** [WP-API/live-settings#1]
One thought that crossed my mind when going through a design review was how quick the saving was. Should it really be per character or would it on tab/moving out of input be better? Maybe the visual appearing after that is better, even if it does save after each character. The issue is there are flashes of the 'Saved!' as it happens, which is distracting.

**Comment by jwold** [comment]
Great point. I thought through this a bit, tested out the plugin, and sketched up the current state in WordPress today, the current state with the Live Settings plugin activated, and my proposal (based heavily on Medium) for how we could change it:

First, here's the current states:

<img width="1119" alt="thing1" src="https://user-images.githubusercontent.com/3411173/29754391-c81e23f8-8b38-11e7-9a5c-825652ab8662.png">

Then, here's the proposal for how we could change things:

<img width="1366" alt="thing2" src="https://user-images.githubusercontent.com/3411173/29754443-d905dc78-8b39-11e7-9cad-fd4e6cbce805.png">

So basically we're taking the "Saved √" notice away from individual elements and putting them into one location at the top of the page that gives a persistent update on whether the page is saved, saving, or unsaved. 

There's one problem with this though, if I scroll down I won't see the current saved state. 

So, here's another proposal moving the save status above into the top bar:

<img width="1131" alt="thing3" src="https://user-images.githubusercontent.com/3411173/29754488-0e585198-8b3b-11e7-9913-470f1bba1e89.png">

I think this option probably makes the most sense. Thoughts?
